src/Auth0.OidcClient.Android/ChromeCustomTabsWebView.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Support.CustomTabs;
+
+namespace Auth0.OidcClient
+{
+    public class ChromeCustomTabsWebView : AndroidWebViewBase
+    {
+        protected override void LaunchBrowser(Android.Net.Uri uri)
+        {
+            var builder = new CustomTabsIntent.Builder();
+            var customTabsIntent = builder.Build();
```

Comment:
if you're not going to use the `builder` then you can merge both lines above into 1.

```js
var customTabsIntent = new CustomTabsIntent.Builder().Build();
```